jekyll/_cci2/configuration-reference.md

```diff
@@ -442,6 +452,9 @@ jobs:
       ... // other config
 ```
 
+**Note**: Java, Erlang and any other languages that introspect the `/proc` directory for information about CPU count may require additional configuration to prevent them from slowing down when using the Docker resource classes. Programs with this issue may request 32 CPU cores and run slower than they would when requesting one core. Users of languages with this issue should pin their CPU count to their guaranteed CPU resources.
```

Comment:
"Users of languages with this issue should pin their CPU count to their guaranteed CPU resources." How do they do that?

Reply:
Sorry just realised this text is moved rather than added in this PR